### ISSUE TITLE
One question about dataset.

### ISSUE THREAD

**Issue by peoplelu** [Vision-CAIR/3DCoMPaT-v2#3]
After download the dataset， I met this question：
![O9}ER950FCLUAV18 5AL42P](https://github.com/Vision-CAIR/3DCoMPaT-v2/assets/82745823/98849d3d-8aaa-438d-8a02-ce38244f0a59)

**Comment by HabibSlim** [comment]
Hello @peoplelu 
Can you clarify what are you trying to do?
The code you're showing is from the PointNeXT dataloader.

Kindly refer to the documentation to see how to iterate over pointclouds using our dataloaders:

[https://3dcompat-dataset.org/doc/loaders-3D-PC.html](https://3dcompat-dataset.org/doc/loaders-3D-PC.html)

**Comment by peoplelu** [comment]
I tried python3 download.py --n-comp compat_010 --modality 3D_PC --split train--semantic-level coarse --outdir ./3DCoMPaT_3D_PC/ to download the point cloud dataset. Then I tried PointNeXt/test_seg.sh and found this error.

**Comment by HabibSlim** [comment]
Noted!

The dataloaders in the PointNeXT "example" folder are given are only provided indicatively, and will not be compatible with the dataset since we have stripped the test set labels for the challenge we're organizing at CVPR.

To get out-of-the-box operating dataloaders, please refer to:

[https://3dcompat-dataset.org/doc/loaders-3D-PC.html](https://3dcompat-dataset.org/doc/loaders-3D-PC.html)

And:

[https://github.com/Vision-CAIR/3DCoMPaT-v2/blob/main/loaders/3D/compat3D_PC.py](https://github.com/Vision-CAIR/3DCoMPaT-v2/blob/main/loaders/3D/compat3D_PC.py)

I have clarified this last point in aa6b1205399b2483c0b50fbfa42ddb68936e4753
